crates/cheatcodes/src/script.rs

```diff
@@ -174,6 +236,7 @@ fn broadcast(ccx: &mut CheatsCtxt, new_origin: Option<&Address>, single_call: bo
         original_origin: ccx.ecx.env.tx.caller,
         depth: ccx.ecx.journaled_state.depth(),
         single_call,
+        delegation,
```

Comment:
hmm, why do we need to keep track of delegation here?

Reply:
Removed, thanks